docs/using/authenticating-to-github.md

```diff
@@ -0,0 +1,35 @@
+# Authenticating to GitHub
+
+## How to sign in to GitHub
+
+1. Open the **GitHub** window by going to the top level **Window** menu and selecting **GitHub**, as shown below.
+
+    <img src="images/github-menu-item.png" alt="GitHub menu item in the Window menu" width="500px"/>
+
+1. Click the **Sign in** button at the top right of the window.
+
+    <img src="images/github-sign-in-button.png" alt="GitHub menu item in the Window menu" width="500px"/>
+
+1. In the **Authenticate** dialog, enter your username or email and password
+
+   <img src="images/github-authenticate.png" alt="GitHub menu item in the Window menu" width="350px"/>
+
+      If your account requires Two Factor Authentication, you will be prompted for your auth code.
+
+      <img src="images/github-two-factor.png" alt="GitHub menu item in the Window menu" width="350px"/>
+
+You will need to create a GitHub account before you can sign in, if you don't have one already.
+
+- For more information on creating a GitHub account, see "[Signing up for a new GitHub account](https://help.github.com/articles/signing-up-for-a-new-github-account/)".
+
+### Personal access tokens
+
+If the sign in operation above fails, you can manually create a personal access token and use it as your password.
+
+The scopes for the personal access token are: `user`, `repo`.
+- *user* scope: Grants access to the user profile data. We currently use this to display your avatar and check whether your plans lets you publish private repositories.
+- *repo* scope: Grants read/write access to code, commit statuses, invitations, collaborators, adding team memberships, and deployment statuses for public and private repositories and organizations. This is needed for all git network operations (push, pull, fetch), and for getting information about the repository you're currently working on.
```

Comment:
We should note that some versions ask for `gist` and `write:public_key`, because that's what the asset store version asks for and we haven't shipped any releases that don't

Reply:
Handled in #828